docs/development.md

```diff
@@ -22,24 +22,63 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 
 ### With Eclipse
 
-* To start the dev server, right-click on the project folder and choose `Run As → Web Application`.<br>
-  After some time, you should see this message (or similar) on the Eclipse console: `Dev App Server is now running`.
-  The dev server URL will be given at the console output, e.g `http://localhost:8888`.
+#### Starting the dev server
 
-* To stop the dev server, click the "Terminate" icon on the Eclipse console.
+Right-click on the project folder and choose `Run As → Web Application`.<br>
+After some time, you should see this message (or similar) on the Eclipse console: `Dev App Server is now running`.
+The dev server URL will be given at the console output, e.g `http://localhost:8888`.
 
-### Outside Eclipse
+#### Stopping the dev server
+
+Click the "Terminate" icon on the Eclipse console.
+
+### With IntelliJ
+
+> If this is your first time running the dev server, you will need to set up the required `Run Configuration`.
+
+#### Set up the Run Configuration
+
+1. Go to `File → Project Structure...`.
+1. Under `Artifacts → Gradle : <your-project-name>.war (exploded)`, check `Include in project build`.
+1. Click `OK`.
+1. Got to `Run → Edit Configurations...`.
+1. Click `+ → Google AppEngine Dev Server`.
+1. Name it `Dev Server`.
+1. Click `Configure` next to `Application server`.
+1. Click `+ → ...`. Select the App Engine SDK you downloaded in Step 3 of the [Setting up a development environment](settingUp.md) guide.
+1. Under `Open browser`, uncheck `After launch`.
+1. Set the `JRE` to `1.7`.
+1. Set the `Port` to `8888`.
+1. Under `Before launch`, click `+ → Run Gradle task`.
+1. Select the local repository as the Gradle project and type "assemble" into the `Tasks` field.
+1. Click `OK`.
+1. Move the item you just created above "Build".
+1. Click `OK`.
+
+#### Starting the dev server
+
+Go to `Run → Play` and select `Dev Server` in the pop-up box.
```

Comment:
This is strange. Is this even after configuring the JRE? I have Java 8 on my computer for another IntelliJ project but didn't run into this issue though.